spec/autocomplete-manager-integration-spec.coffee

```diff
@@ -336,7 +335,7 @@ describe 'Autocomplete Manager', ->
               charMatch = editorView.querySelector('.autocomplete-plus li span.word .character-match')
               expect(word.textContent).toBe 'ab(c)c'
               expect(charMatch.textContent).toBe 'c'
-              expect(charMatch.parentNode).toHaveClass 'word'
+              expect(charMatch.parentNode.parentNode).toHaveClass 'word'
```

Comment:
This makes the test invalid. The entire point of this test is this line. In this new code, the first `c`, the snippet, will be highlighted, when the second `c` is the one that should be highlighted.